site/content/en/docs/components/deprecated/plank.md

```diff
@@ -1,17 +1,20 @@
 ---
-title: "prow/plank/README.md"
+title: "Plank"
```

Comment:
@listx I am not sure whether Plank is deprecated or not.
If not, then I will move this to another category.

Reply:
It is deprecated. For the record, prow.k8s.io does not use it (it uses the prow-controller-manager binary instead).